pipeline/terraform/modules/stack/release_uris.tf

```diff
@@ -7,17 +7,13 @@ locals {
   repo_urls = [for repo_url in data.aws_ecr_repository.service.*.repository_url : "${repo_url}:env.${var.release_label}"]
   image_ids = zipmap(local.services, local.repo_urls)
 
```

Comment:
Why are the non-router images removed here? I think they are still needed, the lambdas use the container images they contain.